.editorconfig

```diff
@@ -1,901 +1,901 @@
-[*]
-charset = utf-8
-end_of_line = crlf
```

Comment:
this line is the change, everything else is caused by the removal of the `^m` windows end of line character